src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -578,13 +597,23 @@ namespace NServiceBus
         [System.ObsoleteAttribute("Please use `StartAsync()` instead. Will be removed in version 7.0.0.", true)]
         public static NServiceBus.IBus Start(this NServiceBus.IStartableBus bus) { }
     }
+    public interface IStartableEndpoint
+    {
+        System.Threading.Tasks.Task<NServiceBus.IStoppableEndpoint> StartAsync();
+    }
+    public interface IStoppableEndpoint
```

Comment:
Yea, IMO adding `Instance` will fix this since a instance is sort of by definition "stoppable"?